apps/app/src/app/pages/pages/sessions/pages/session-detail/components/tasks-list/tasks-list.component.spec.ts

```diff
@@ -4,7 +4,7 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { UiModule } from '@armonik.admin.gui/ui';
 import { ClarityModule } from '@clr/angular';
 import { TranslateModule } from '@ngx-translate/core';
-
+import { TaskStatusFilterComponent } from '../../../../../../../shared';
```

Comment:
Are you sure it is not '../../../../../../../../shared' ?
/s

Reply:
done